support-frontend/assets/pages/[countryGroupId]/checkout.tsx

```diff
@@ -250,6 +270,100 @@ export function Checkout() {
 								/>
 							</BoxContents>
 						</Box>
+						<form
+							action="/contribute/recurring/create"
+							method="POST"
+							onSubmit={(event) => {
+								event.preventDefault();
+								const form = event.currentTarget;
+								const formData = new FormData(form);
+
+								/**
+								 * The validation for this is currently happening on the client side form validation
+								 * So we'll assume strings are not null.
+								 * see: https://developer.mozilla.org/en-US/docs/Learn/Forms/Form_validation
+								 */
+								const data = {
+									firstName: formData.get('firstName') as string,
+									lastName: formData.get('lastName') as string,
+									email: formData.get('email') as string,
+									product: formData.get('product') as string,
+									ratePlan: formData.get('ratePlan') as string,
+									currency: formData.get('currency') as string,
+								};
+								console.info('Posting data', data);
+
+								// Currently support-workers doesn't understand the new shape, and we also need to
+								// add payment data, so for now, just abort.
+								return false;
+							}}
+						>
+							<input type="hidden" name="product" value={query.product} />
+							<input type="hidden" name="ratePlan" value={currentCurrencyKey} />
+							<input type="hidden" name="currency" value={currentCurrencyKey} />
```

Comment:
We could just read the `query[name]` here , but it felt like having one way to get the data from the form was consistent and maintained the use of standard API e.g. `input => FormData`